pintc/tests/interfaces/simple.pnt

```diff
@@ -52,10 +52,6 @@ predicate Simple {
     state x = FooInstance::storage::x;
     state y = FooInstance::storage::y;
     state z = BarInstance::storage::z;
-    constraint x == 0;
```

Comment:
I don't think these are meant to be removed 🤔

Reply:
Are you saying for all of the constraints that I put in for PR #833, or just this particular instance in `simple.pnt`?

My understanding is that we don't want these because the constraints were added to avoid optimising out the dead states. I added these in commit 6780d484884a2e7e61d446aa4c3c13da07ce0cbe and set them to a nontrivial value in commit 84e9ca9e46e0300851a013cbfd415b2cf72ee92b.